content/en/serverless/custom_metrics/_index.md

```diff
@@ -198,51 +182,30 @@ For example:
 }
 ```
 
-**Note:** These custom metrics are submitted as [distributions](#custom-metrics). If you were previously submitting custom metrics another way, [consult the documentation on the implications of upgrading to distributions](#understanding-distribution-metrics).
-
-[1]: /serverless/forwarder/
 {{< /programming-lang >}}
 {{< /programming-lang-wrapper >}}
 
-### Tagging custom metrics
-
-You should tag your custom metrics when submitting them with the [Datadog Lambda Library][7]. Use the [Distribution Metrics][3] page to [customize the set of tags][8] applied to your custom metrics.
+## Third-party libraries
 
-To add Lambda resource tags to your custom metrics, set the parameter `DdFetchLambdaTags` to `true` on the Datadog forwarder CloudFormation stack.
-
-### Understanding distribution metrics
-
-With distribution metrics, you select the aggregation when graphing or querying it instead of specifying it at submission time.
-
-If you previously submitted custom metrics from Lambda without using one of the Datadog Lambda Libraries, you'll need to start instrumenting your custom metrics under **new metric names** when submitting them to Datadog. The same metric name cannot simultaneously exist as both distribution and non-distribution metric types.
-
-To enable percentile aggregations for your distribution metrics, consult the [Distribution Metrics][8] page.
-
-## Other submission methods
-
-### Running in a VPC
-
-The Datadog Lambda Library requires [access to the public internet][9] to submit custom metrics **synchronously**. If your Lambda function is associated with a VPC, ensure that it is instead submitting custom metrics **asynchronously** or that your function can reach the public internet.
-
-### Third-party libraries
-
-There are a number of open source libraries that make it easy to submit custom metrics to Datadog. However, many have not been updated to use [Distribution metrics][3], which are optimized for Lambda. Distribution metrics allow for server-side aggregations independent of a host or locally-running [agent][10]. In a serverless environment where there is no agent, Distribution metrics give you flexible aggregations and tagging.
+There are a number of open source libraries that make it easy to submit custom metrics to Datadog. However, many have not been updated to use [Distribution metrics][2], which are optimized for Lambda. Distribution metrics allow for server-side aggregations independent of a host, and give you flexible aggregations and tagging.
```

Comment:
I'm not sure this section makes sense to include unless we are recommending some third-party libraries and linking to which third party libraries make this easy (which I'm not sure makes sense to do?)

Reply:
Yeah, agree, will remove.